parts/kubernetesmastervars.t

```diff
@@ -75,7 +75,8 @@
     "masterInternalLbID": "[resourceId('Microsoft.Network/loadBalancers',variables('masterInternalLbName'))]",
     "masterInternalLbIPConfigName": "[concat(variables('orchestratorName'), '-master-internal-lbFrontEnd-', variables('nameSuffix'))]",
     "masterInternalLbIPConfigID": "[concat(variables('masterInternalLbID'),'/frontendIPConfigurations/', variables('masterInternalLbIPConfigName'))]",
-    "masterInternalLbIp": "[concat(variables('masterFirstAddrPrefix'), add(variables('masterCount'), int(variables('masterFirstAddrOctet4'))))]",
+    "masterInternalLbIPOffset": 10,
```

Comment:
Update the commit message with the most important reason: "having a consistent LB IP address, makes troubleshooting customer incidents easier".